src/porting.cpp

```diff
@@ -532,12 +532,13 @@ void initializePaths()
 		dstream << "WARNING: Could not determine bundle resource path" << std::endl;
 	}
 	CFRelease(resources_url);
-	path_user = std::string(getenv("HOME")) + "/Library/Application Support/" + PROJECT_NAME;
+
+	path_user = std::string(getenv("HOME")) + "/Library/Application Support/" + lowercase(PROJECT_NAME);
 
 	#else // FreeBSD, and probably many other POSIX-like systems.
```

Comment:
This is a perfect example of why I think the # part of the directive should be at the start of the line and the tab after the #

CC. @kwolekr